pages/apis/rest_api/organization/members.md

```diff
@@ -0,0 +1,62 @@
+# Organization members API
+
+The organization members API allows users to view all members of an organization.
+
+## Organization member data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>id</code></th><td>UUID of the user</td></tr>
+  <tr><th><code>name</code></th><td>Name of the user</tr>
+  <tr><th><code>email</code></th><td>Email of the user</td></tr>
+</tbody>
+</table>
+
+## List organization members
+
+Returns a list of an organization's members
+
+```bash
+curl --request GET \
+  --header 'Authorization: Bearer bkua_xxx' \
```

Comment:
There's not one pattern, but I think we should go with the one on the [Overview page](https://buildkite.com/docs/apis):

```suggestion
  --header 'Authorization: Bearer $TOKEN' \
```